src/js/measurements.ts

```diff
@@ -48,10 +48,23 @@ const _patchStartTransaction = (
     transactionContext: TransactionContext,
     customSamplingContext?: CustomSamplingContext
   ): Transaction {
+    if (!transactionContext.op) {
+      transactionContext.op = 'default';
```

Comment:
M: I'm wondering if this should live here, the file is about measurements, is the reason that we only patch the `startTransaction` and `startChild` here? Can we move this patching else where?

Reply:
I'll open a new issue for the refactoring.